app/src/main/res/values/themes.xml

```diff
@@ -17,7 +17,7 @@
         <item name="headerTextColor">@color/white</item>
 
         <!-- Progress Bar -->
-        <item name="progressBackground">@color/black</item>
+        <item name="progressBackground">#CC4b4b4b</item>
```

Comment:
Can you use a color value for this (and while you're at it, update the primary/secondary ones) so we can theme the progress bar in the future?